dev/com.ibm.ws.cdi.jndi_fat/fat/src/com/ibm/ws/cdi/jndi/lookup/BeanManagerLookupServlet.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.jndi.lookup;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Set;
+
+import javax.enterprise.inject.spi.Bean;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.enterprise.inject.spi.CDI;
+import javax.inject.Inject;
+import javax.naming.InitialContext;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@WebServlet("/")
+public class BeanManagerLookupServlet extends FATServlet {
+
+    private static final long serialVersionUID = 8549700799591343964L;
+
+    @Inject
+    BeanManager injectedBeanManager;
+
+    @Inject
+    MyBean myRequestScopedBean;
+
+    private BeanManager getBeanMangerViaJNDI() throws Exception {
+        return (BeanManager) new InitialContext().lookup("java:comp/BeanManager");
+    }
+
+    public void testBeanManager(String beanManagerDescription, BeanManager beanManager) {
+        assertNotNull(beanManagerDescription + " was null", beanManager);
```

Comment:
Missing `@Test` annotation

Reply:
doh!

Reply:
ah, no, that is a common test method, used by the other tests ... perhaps it should not be public?

Reply:
Oh, so it is, yes make it non-public